tests/baselines/reference/spreadOfParamsFromGeneratorMakesRequiredParams.errors.txt

```diff
@@ -11,5 +11,4 @@ tests/cases/compiler/spreadOfParamsFromGeneratorMakesRequiredParams.ts(6,1): err
     
     call(function* (a: 'a') { }); // error, 2nd argument required
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2554: Expected 2 arguments, but got 1.
-!!! related TS6210 tests/cases/compiler/spreadOfParamsFromGeneratorMakesRequiredParams.ts:3:5: An argument for 'args' was not provided.
\ No newline at end of file
+!!! error TS2345: Argument of type '[]' is not assignable to parameter of type 'never'.
```

Comment:
So this swapped from an arity error to an assignability error because...? I would have assumed the arity error was preferable here. As written, there's no `never` or `[]`, so I'm really not sure what I'd need to change to fix this error.

Reply:
The `never` comes from `Parameters<T>` when `T` isn't a function type. Neither error is particularly great, but the new error message is in some sense more correct. And, if you add a second argument in an attempt to fix the old error message, you're just greeted with the new error message.

Reply:
Ahh.... But a generator _is_ a function type. You call it and get back an iterator, just like how you call an async function and get a promise. This error is straight wrong, then.